Verso Books.js

```diff
@@ -0,0 +1,313 @@
+				"libraryCatalog": "Verso Books",
+				"publisher": "Verso Books",
+				"shortTitle": "China in One Village",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.versobooks.com/books/3814-capitalism-and-the-camera",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Capitalism and the Camera: Essays on Photography and Extraction",
+				"creators": [
+					{
+						"firstName": "Kevin",
+						"lastName": "Coleman",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Daniel",
+						"lastName": "James",
+						"creatorType": "editor"
+					}
+				],
+				"date": "2021-05",
+				"ISBN": "9781839760808",
+				"abstractNote": "Photography was invented between the publication of Adam Smith’s The Wealth of Nations and Karl Marx and Friedrich Engels’s The Communist Manifesto. Taking the intertwined development of capitalism and the camera as their starting point, the essays collected here investigate the relationship between capitalist accumulation and the photographic image, and ask whether photography might allow us to refuse capitalism’s violence—and if so, how?\n\nDrawn together in productive disagreement, the essays in this collection explore the relationship of photography to resource extraction and capital accumulation, from 1492 to the postcolonial; the camera’s potential to make visible critical understandings of capitalist production and society, especially economies of class and desire; and the ways the camera and the image can be used to build cultural and political counterpublics from which a democratic struggle against capitalism might emerge.\n\nWith essays by Ariella Aïsha Azoulay, Siobhan Angus, Kajri Jain, Walter Benn Michaels, T. J. Clark, John Paul Ricco, Blake Stimson, Chris Stolarski, Tong Lam and Jacob Emery.",
+				"libraryCatalog": "Verso Books",
+				"publisher": "Verso Books",
+				"shortTitle": "Capitalism and the Camera",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.versobooks.com/books/1508-yellow-peril",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Yellow Peril!: An Archive of Anti-Asian Fear",
+				"creators": [
+					{
+						"firstName": "John Kuo Wei",
+						"lastName": "Tchen",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Dylan",
+						"lastName": "Yeats",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014-01",
```

Comment:
This date changed for some reason, so the test fails now.

Reply:
Thanks for pointing it out! It might be caused by `Zotero.Utilities.strToDate`. I am guessing that function was based on JS native date, with January starting from 0, cause it gave me 5 for June, so I later switched to `strToISO`. I updated the test.

Reply:
Thanks! Yeah, `strToISO` is almost always what you want to use in a translator.